interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -30,7 +30,7 @@ angular.module('adage.analyze.sampleBin', [
 function($log, $cacheFactory, $q, Sample, Activity, Signature, SignatureSet,
 MathFuncts, errGen) {
   var SampleBin = {
-    selectedMlModel: {
+    mlModelInfo: {
```

Comment:
Does it make more sense to keep track of this information here or to just inject the `MlModelTracker` factory as a dependency to this factory and grab the information from there?